dot/network/service.go

```diff
@@ -670,6 +669,9 @@ func (s *Service) startPeerSetHandler() {
 
 func (s *Service) processMessage(msg peerset.Message) {
 	peerID := msg.PeerID
+	if peerID == "" {
+		logger.Errorf("found empty peer id in peerset message")
```

Comment:
should we return here?

Reply:
No, processMessage is used to execute network tasks. If peer id is empty, that means we cannot execute that particular task. We should continue to process other tasks. If we get an empty message, that mean we have written wrong code (somewhere we are sending mesage without peerid).

This check is a precautionary measure.

Reply:
> If peer id is empty, that means we cannot execute that particular task. We should continue to process other tasks.

What do you mean by 'particular task'? In my view, `processMessage(msg)` is that particular task, and we should return if the peer id is empty right?

Reply:
Oh yes, my bad! Thanks for spotting this! Also, Fixed!